src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -107,6 +107,33 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice web config hostname', 'webapp', help="webapp name", completer=get_resource_name_completion_list('Microsoft.Web/sites'), id_part='name')
 register_cli_argument('appservice web config hostname', 'name', arg_type=name_arg_type, completer=get_hostname_completion_list, help="hostname assigned to the site, such as custom domains", id_part='child_name')
 
+database_types = ['SqlAzure', 'MySql']
+frequency_units = ['Hour', 'Day']
+
+register_cli_argument('appservice web config backup create', 'storage_account_url', help='SAS URL for the blob storage container')
+register_cli_argument('appservice web config backup create', 'backup_name', help='If specified, backup blob will be named <backup-name>.zip')
+register_cli_argument('appservice web config backup create', 'database_name', help='Name of the database to backup')
+register_cli_argument('appservice web config backup create', 'database_connection_string', help='Connection string for the database to backup')
+register_cli_argument('appservice web config backup create', 'database_type', help='Type of database being backed up', **enum_choice_list(database_types))
+
+register_cli_argument('appservice web config backup update', 'storage_account_url', help='SAS URL for the blob storage container')
+register_cli_argument('appservice web config backup update', 'schedule_frequency', help='Numerical frequency for how often to backup')
+register_cli_argument('appservice web config backup update', 'schedule_frequency_unit', help='Unit for how often to backup', **enum_choice_list(frequency_units))
```

Comment:
I'll go with adding parsing for simple strings. The unit and number being split into separate components are a complexity of our REST API, but we should hide that from users where possible.